docs/runbooks/cloud9/cloud9_init.sh

```diff
@@ -8,9 +8,8 @@ function main() {
 }
 
 function install_tools() {
-  #update to 9.6 - to match rds version deployed
-  #this may need further adjusting as the environments may have a newer one e.g. 10.3
-  sudo yum install postgresql96 jq -y
+  sudo amazon-linux-extras install postgresql10 jq -y
```

Comment:
Do you need jq in this line too?

Reply:
i did have, that didn't work due to being not on the extras topic. it's a line below instead.

Reply:
Ah i see it - i id make a change locally 🙈